modules/tibc/apps/nft_transfer/keeper/relay.go

```diff
@@ -180,14 +196,19 @@ func (k Keeper) refundPacketToken(ctx sdk.Context, data types.NonFungibleTokenPa
 
 	if data.AwayFromOrigin {
 		// unlock
-		if err := k.nk.TransferOwner(ctx, data.Class, data.Id, "", data.Uri, "",
+		if err := k.nk.TransferOwner(ctx, data.Class, data.Id, DoNotModify, DoNotModify, DoNotModify,
 			k.GetNftTransferModuleAddr(types.ModuleName), sender); err != nil {
 			return err
 		}
-
 	} else {
 		// mintNFT
-		if err := k.nk.MintNFT(ctx, data.Class, data.Id, "", data.Uri, "", sender); err != nil {
+		// Corresponding to burnNft, because the mintRestricted attribute of denom generated by any cross-chain nft is true,
+		// so to re-mint nft, you must first mintnft to the module account, and then transfer the nft ownership to the sender account
+		if err := k.nk.MintNFT(ctx, data.Class, data.Id, "", data.Uri, "", k.GetNftTransferModuleAddr(types.ModuleName)); err != nil {
+			return err
+		}
+
+		if err := k.nk.TransferOwner(ctx, data.Class, data.Id, DoNotModify, DoNotModify, DoNotModify, k.GetNftTransferModuleAddr(types.ModuleName), sender); err != nil {
```

Comment:
Define `moudleAddr := k.GetNftTransferModuleAddr(types.ModuleName)` as the method `SendNftTransfer`.